### DISCUSSION TITLE
minimig

### DISCUSSION THREAD

**Discussion by JJDSNT** [MiSTle-Dev/MiSTeryNano#16] [General]
From the MiSTeryNano experience would be possible to evaluate how hard would be to port the minimig core to the tang nano?

i think claude (pistorm) will try 

https://twitter.com/Claude1079/status/1663917904826343432

and maybe the pistorm can also be used with your atari implementation?

Beste regards,

**Comment by harbaum** [comment]
I am pretty sure the Minimig can be ported as the Atari ST and the Amiga are quite similar. Maybe AGA is a little bit too much, but OCS and ECS should IMHO be possible. I am using the cycle exact fx68k CPU core. Using the TG68k instead would also save some FPGA space. IMHO AGA with the fx68k doesn't make too much sense, anyway, as the AGA machines all had >= 68020 CPUs. The tg68k can do this to a certain extent.

Claude is running the Minimig chipset in the TN20k without CPU. But if he actually has such a running setup up and running i'd expect it to actually be a minor problem to include a CPU.

And if he has a PiStorm acting as a CPU for an Amiga running inside the FPGA it is likely possible to do the same with my core as well. But I personally have no interest in this. So this is up to someone else.

**Reply by harbaum** [reply]
For the records: minimig has been ported at http://GitHub.com/harbaum/nanomig
